docs/deploying a sample wildfly application.md

```diff
@@ -0,0 +1,144 @@
+
+<a name="deploying-a-hello-world-wildfly-application"></a>
+## Deploying a Hello World WildFly application
+
+Once you have started Minishift, follow these instructions to deploy a Hello World application using WildFly in OpenShift:
+
+1. Login to your OpenShift account, if you are not already logged in:
+
+        $ oc login https://<minishift ip>:8443 -u developer -p developer
+        Login successful.
+
+        You have one project on this server: "myproject"
+
+        Using project "myproject".
+
+1. (Optional step) The current example uses the default project “myproject”. You can either choose to use “myproject” for deployment of this application or create a fresh project for the same as:
+
+        $ oc new-project <projectname> --display-name="Abc_123"
+        Now using project "projectname" on server "https://192.168.42.232:8443".
+
+        You can add applications to this project with the 'new-app' command. For example, try:
+
+        oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git
+
+        to build a new example application in Ruby.
+
+1. To create a WildFly application you need to use either a [WildFly Imagestream or a template](https://docs.openshift.org/latest/install_config/imagestreams_templates.html). Check the available Imagestreams already registered in your OpenShift Origin cluster by doing one of the following:
+
+        $ oc get is -n openshift
+        NAME         DOCKER REPO                               TAGS                         UPDATED
+        jenkins      172.30.47.201:5000/openshift/jenkins      latest,1                     7 days ago
+        mariadb      172.30.47.201:5000/openshift/mariadb      latest,10.1                  7 days ago
+        mongodb      172.30.47.201:5000/openshift/mongodb      latest,3.2,2.6 + 1 more...   7 days ago
+        mysql        172.30.47.201:5000/openshift/mysql        latest,5.6,5.5               7 days ago
+        nodejs       172.30.47.201:5000/openshift/nodejs       4,0.10,latest                7 days ago
+        perl         172.30.47.201:5000/openshift/perl         5.20,5.16,latest             7 days ago
+        php          172.30.47.201:5000/openshift/php          latest,5.6,5.5               7 days ago
+        postgresql   172.30.47.201:5000/openshift/postgresql   latest,9.5,9.4 + 1 more...   7 days ago
+        python       172.30.47.201:5000/openshift/python       latest,3.5,3.4 + 2 more...   7 days ago
+        ruby         172.30.47.201:5000/openshift/ruby         latest,2.3,2.2 + 1 more...   7 days ago
+        wildfly      172.30.47.201:5000/openshift/wildfly      10.0,9.0,8.1 + 1 more...     7 days ago
+
+  You will notice that the WildFly imagestream already exists in your OpenShift Origin cluster. If a WildFly imagestream is unavailable you can [create an Imagestream](https://docs.openshift.org/latest/dev_guide/managing_images.html) to build your application.
```

Comment:
Agreed, will remove the above sentence, and modify the sentence below too to give a more generic link to OpenShift docs for further information on creating a new application>https://docs.openshift.org/latest/dev_guide/application_lifecycle/new_app.html